ChangeLog.d/MD-X.509-hashing.txt

```diff
@@ -0,0 +1,2 @@
+Features
```

Comment:
Please avoid abbreviations in commit messages, and include relevant information. We adopt the widespread convention of a short summary line (no hard maximum, but ≤70 chars is strongly preferred) followed by a blank line followed by a longer description (if the summary isn't enough). Suggestion:

```
X.509: use PSA for hashing under USE_PSA_CRYPTO

When MBEDTLS_USE_PSA_CRYPTO is enabled, use psa_hash_xxx rather than
mbedtls_md_xxx.
```